packages/scope-manager/src/analyze.ts

```diff
@@ -57,9 +57,9 @@ interface AnalyzeOptions {
    */
   sourceType?: SourceType;
 
+  // TODO - remove this in v8
   /**
-   * Emit design-type metadata for decorated declarations in source.
-   * Defaults to `false`.
+   * @deprecated This option no longer does anything and will be removed in a future major release.
```

Comment:
I came into this confused at seeing as what looked like "breaking" change in a minor. Just making sure nobody else gets confused by that too:

```suggestion
   * @deprecated This option never did what it was intended for and will be removed in a future major release.
```